### PR TITLE
Color Palette

### LINKED ISSUE #6: Define color palette
We must define a color palette to start developing the prototype!

This is a site for trading album stickers, specially for the FIFA World Cup Panini sticker album.

Feel free to improve as you want!

### PR DESCRIPTION
**Pull request by william-monroy**
Issue #6 

I create a file `colors.js` at `/design` folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -118,6 +118,7 @@ Thanks goes to these wonderful people ([emoji key](https://allcontributors.org/d
...
       <td align="center"><a href="https://github.com/ABHISHEK-565"><img src="https://avatars.githubusercontent.com/u/80988197?v=4?s=100" width="100px;" alt="Abhishek singh lodhi"/><br /><sub><b>Abhishek singh lodhi</b></sub></a><br /><a href="https://github.com/trading-album-stickers/trading-album-stickers-front/commits?author=ABHISHEK-565" title="Documentation">📖</a></td>
       <td align="center"><a href="https://github.com/imgourav"><img src="https://avatars.githubusercontent.com/u/72176692?v=4?s=100" width="100px;" alt="Gourav Sharma"/><br /><sub><b>Gourav Sharma</b></sub></a><br /><a href="https://github.com/trading-album-stickers/trading-album-stickers-front/commits?author=imgourav" title="Documentation">📖</a></td>
+      <td align="center"><a href="https://github.com/william-monroy"><img src="https://avatars.githubusercontent.com/u/58092741?v=4?s=100" width="100px;" alt="William Monroy"/><br /><sub><b>William Monroy</b></sub></a><br /><a href="https://github.com/trading-album-stickers/trading-album-stickers-front/commits?author=william-monroy" title="Design">🎨</a></td>
```
Comment (dcruzb) [inline README.md]: This line is added automatically, you don't need to put your name here. It is added by the allcontributiors bot. Could you please remove it?

**Review by dcruzb — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (william-monroy) [reply README.md]: Sure! :octocat:

**Comment by william-monroy** [comment]
@dcbCIn Could you review the changes, please?

**Review by dcruzb — APPROVED** [review]

**Comment by william-monroy** [comment]
Could you add the hacktoberfest labels, please

**Comment by dcruzb** [comment]
@all-contributors please add @william-monroy for design

**Comment by dcruzb** [comment]
They are already on the issue, but since you asked I will add it here too.

**Pull request merged**